site/content/docs/4.6/components/navs.md

```diff
@@ -3,6 +3,7 @@ layout: docs
 title: Navs
 description: Documentation and examples for how to use Boosted's included navigation components.
 group: components
+aliases: "/docs/4.6/components/navbar-orange/"
```

Comment:
Maybe I have misunderstood something but for me you'd rather do this:

- Remove this line in this file because we don't want this redirection
- Rename `navbar-orange.md` in `orange-navbar.md`
- Insert this line in `orange-navbar.md`